### ISSUE TITLE
Whatsapp mobile layout on desktop

### ISSUE THREAD

**Issue by Fuseteam** [alefnode/ubports-apps#3]
currently it uses the same mobile layout on a mobile screen and on a desktop screensize

**Comment by alefnode** [comment]
Yeah, to use on desktop screen you can use morph, i think, but i can try to check if app are in desktop mode or mobile, i have to investigate that

**Comment by Fuseteam** [comment]
i think you can do that with the size of the screen i think
